plugins/modules/entrust_cagw_certificate.py

```diff
@@ -0,0 +1,860 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c), Entrust Datacard Corporation, 2019
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: entrust_cagw_certificate
+author:
+    - Sapna Jain (@jains)
+short_description: Request SSL/TLS certificates with the Certificate Authority Gateway (CAGW) API
+description:
+    - Create, get, and take actions (Hold, Unhold, Revoke certificates) with the Certificate Authority Gateway (CAGW) API.
+    - Requires credentials for calling the CAGW API.
+notes:
+    - C(path) must be specified as the output location of the certificate.
+requirements:
+    - cryptography >= 1.6
+options:
+    force:
+        description:
+            - If force is used, a certificate is requested regardless of whether I(path) points to an existing valid certificate.
+        type: bool
+        default: False
+    path:
+        description:
+            - The destination path for the generated certificate as a PEM encoded cert.
+            - If there is already an Entrust certificate at this location, it will be replaced always.
+            - If enrollment_format is PKCS12 then it will have Base64 encoded PKCS12 body
```

Comment:
```suggestion
            - If O(enrollment_format=PKCS12) then it will have Base64 encoded PKCS12 body.
```
1. Every paragraph should consist out of complete sentences (usually a period at the end).
2. Check out https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation and https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation.